### PR TITLE
Allow i18n for the extension

### LINKED ISSUE #31: Allow translation of the extension
It seems like an oversight to have the extension support non-English wikis, without providing for translations of the extension itself.

### PR DESCRIPTION
**Pull request by Dianliang233**
Fixes #31

I haven't finished marking the guide page yet, so this is still a draft.

After merging, it's probably best to set up a translation service like Crowdin to help crowdsource translations.

### CODE REVIEW

File: scripts/content-banners.js [hunk 1]
```diff
@@ -183,24 +183,29 @@ function displayRedirectBanner(newUrl, id, destinationName, destinationLanguage,
...
   } else {
-    bannerText.textContent = 'There is ' + descriptor + ' wiki covering this topic!';
+    bannerText.textContent = extensionAPI.i18n.getMessage('bannerText', [descriptor]);
   }
   let bannerWikiLink = document.createElement('a');
   bannerWikiLink.classList.add('indie-wiki-banner-link');
   bannerText.appendChild(bannerWikiLink);
   bannerWikiLink.href = newUrl;
-  bannerWikiLink.textContent = 'Visit ' + destinationName + ' →';
+  bannerWikiLink.textContent = extensionAPI.i18n.getMessage('bannerText', [destinationName]);
```
Comment (KevinPayravi) [inline scripts/content-banners.js:208]: This should be `bannerVisit` instead of `bannerText`

File: pages/guide/index.html [hunk 2]
```diff
@@ -5,221 +5,165 @@
...
-          <b><span aria-hidden="true">🌐</span> Redirect non-English Fandom wikis to
-            indie English wikis there is no same-lang wiki</b>
+          <b><span aria-hidden="true">🌐</span>&nbsp;<span data-msg="settingsCrossLanguage"></span></b>
           <br />
-          By default, Indie Wiki Buddy will keep users within the same language
-          (e.g. only redirecting Spanish Fandom wikis to Spanish indie wikis).
-          However, multilingual speakers may want to also use the many independent English wikis that are available.
-          Enabling this option will redirect from non-English Fandom wikis
-          to independent English wikis when one isn't available in the same language.
+          <span data-msg="settingsCrossLanguageHelp1"></span>
```
Comment (KevinPayravi) [inline pages/guide/index.html:52]: Should be `guideCrossLanguageHelp1`

File: pages/guide/index.html [hunk 3]
```diff
@@ -5,221 +5,165 @@
...
           <br /><br />
-          Unfortunately this feature does not currently work for Fextralife wikis.
+          <span data-msg="settingsCrossLanguageHelp2"></span>
```
Comment (KevinPayravi) [inline pages/guide/index.html:54]: Should be `guideCrossLanguageHelp1`

**Review by KevinPayravi — APPROVED** [review]
Thank you! 🎉 Very excited for us to have internationalization support. Will get us set up on Crowdin (or similar) soon.

**Comment by fr0r** [comment]
> Thank you! 🎉 Very excited for us to have internationalization support. Will get us set up on Crowdin (or similar) soon.

Hey! Any updates about it? Still interested in helping out :)

**Pull request merged**